go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -254,12 +242,8 @@ func TestScripts(t *testing.T) {
 		skipped = append(skipped,
 			// Different error output for primary key error
 			"failed statements data validation for INSERT, UPDATE",
```

Comment:
Does your change below not address this discrepancy? Should be simple to make the two formats match their string output for this errror

Reply:
there was another error that failed this test. will circle back